bolt-driver/src/main/java/org/neo4j/ogm/drivers/bolt/request/BoltRequest.java

```diff
@@ -144,34 +146,13 @@ public void close() {
     private StatementResult executeRequest(Statement request) {
         BoltTransaction tx;
         try {
-
             Map<String, Object> parameterMap = mapper.convertValue(request.getParameters(), MAP_TYPE_REF);
             LOGGER.info("Request: {} with params {}", request.getStatement(), parameterMap);
 
-            if (transactionManager.getCurrentTransaction() == null) {
-                org.neo4j.ogm.transaction.Transaction autoCommitTx = transactionManager.openTransaction();
-                tx = (BoltTransaction) autoCommitTx;
-                StatementResult statementResult = tx.nativeBoltTransaction().run(request.getStatement(), parameterMap);
-                tx.commit();
-                tx.close();
-                return statementResult;
-            }
             tx = (BoltTransaction) transactionManager.getCurrentTransaction();
             return tx.nativeBoltTransaction().run(request.getStatement(), parameterMap);
-        } catch (CypherException | ConnectionException ce) {
-            throw ce;
-        } catch (ClientException ce) {
-            tx = (BoltTransaction) transactionManager.getCurrentTransaction();
-            if (tx != null) {
-                tx.rollback();
-            }
+        } catch (ClientException | DatabaseException | TransientException ce) {
```

Comment:
the purpose here was just to do some exception translation, and indicate there was an error while executing the request (and that the tx has been rolled back server side).
The idea is that when exception handling code receives a CypherException, it knows the only option is to rollback (this is probably worth adding to the docs).
I initially started to change exceptions but finally left it as a separate piece of work because 
- not sure we can do much better
- we should agree on the exception strategy beforehand